cairo/tests/ethereum/cancun/test_trie.py

```diff
@@ -101,6 +102,7 @@ def test_nibble_list_to_compact(self, cairo_run, x, is_leaf: bool):
         )
 
     @given(x=nibble.filter(lambda x: len(x) != 0), is_leaf=...)
+    @settings(verbosity=Verbosity.quiet)
```

Comment:
why are we using quiet verbosity?

we can directly replace in CI profile in conftest quiet verbosity

Reply:
quiet for those that contain the cairo_program fixture only, so that the full program is not displayed even in verbose settings (this is what causes the logs to be bloated in CI)